modules/aolBidAdapter.js

```diff
@@ -373,6 +374,11 @@ export const spec = {
       params.us_privacy = consentData.uspConsent;
     }
 
+    if (consentData.gppConsent && consentData.gppConsent.gppString) {
```

Comment:
why not mirror the logic below from yahoo?

Reply:
this is a legacy adapter sending the requests to some legacy endpoints as GET requests. The POST openRTB is not officially supported. We don't apply the openRTB logic here.

Reply:
The only current place you can expect to find the gpp string is in the location yahoo falls back to based on publisher setConfigs.

Reply:
I added the fallback support similar to the yahoo adapter logic.